hibernate-core/src/test/java/org/hibernate/orm/test/dialect/HANADialectTestCase.java

```diff
@@ -53,7 +53,7 @@ public void testSqlGeneratedForIdentityInsertNoColumns() {
 					catch (MappingException e) {
 						MatcherAssert.assertThat(
 								e.getMessage(),
-								is( "The INSERT statement for table [EntityWithIdentity] contains no column, and this is not supported by [" + HANAColumnStoreDialect.class.getName() + "]" )
+								startsWith( "The INSERT statement for table [EntityWithIdentity] contains no column, and this is not supported by [" + HANAColumnStoreDialect.class.getName() )
```

Comment:
I think you might be missing the final "]" ?

Reply:
With the new log message the version might change. Isn't it enough to test the beginning of the message?
I guess I can use a regex

Reply:
Ah, never mind, did not look at what you were actually doing, my bad :facepalm: